tests/manage/mcg/test_host_node_failure.py

```diff
@@ -0,0 +1,202 @@
+import logging
```

Comment:
thinking maybe to either put this test file under tests/manage/mcg or keep the current location but just rename with a name that reflects it's a noobaa related test file

Reply:
@ebenahar yes, this test file is already under `tests/manage/mcg`. Do we still need to rename it?